appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/AbstractSingletonContainer.java

```diff
@@ -634,46 +653,48 @@ public boolean userTransactionMethodsAllowed(ComponentInvocation inv) {
     * Check if the given EJBObject/LocalObject has been removed.
     * @exception NoSuchObjectLocalException if the object has been removed.
     */
+    @Override
     protected void checkExists(EJBLocalRemoteObject ejbObj)
     {
         // Doesn't apply to Singletons
     }
 
+    @Override
     protected void afterBegin(EJBContextImpl context) {
         // Singleton SessionBeans cannot implement SessionSynchronization!!
         // EJB2.0 Spec 7.8.
     }
 
+    @Override
     protected void beforeCompletion(EJBContextImpl context) {
         // Singleton SessionBeans cannot implement SessionSynchronization!!
         // EJB2.0 Spec 7.8.
     }
 
+    @Override
     protected void afterCompletion(EJBContextImpl ctx, int status) {
         // Singleton SessionBeans cannot implement SessionSynchronization!!
     }
 
     // default
+    @Override
     public boolean passivateEJB(ComponentContext context) {
         return false;
     }
 
+    /**
+     * @deprecated not called and not used in Payara 5
+     */
+    @Deprecated
     // default
-    public void activateEJB(Object ctx, Object instanceKey) {}
-
-/** TODO
-    public void appendStats(StringBuilder sbuf) {
-	sbuf.append("\nSingletonContainer: ")
-	    .append("CreateCount=").append(statCreateCount).append("; ")
-	    .append("RemoveCount=").append(statRemoveCount).append("; ")
-	    .append("]");
+    public void activateEJB(Object ctx, Object instanceKey) {
```

Comment:
Warning - there is just one method with same name but with three arguments (in other classes). But this one with two is not used.